pkg/eventpersisterdwh/persister/evaluation_event.go

```diff
@@ -87,6 +87,8 @@ func (w *evalEvtWriter) Write(
 			case *eventproto.EvaluationEvent:
 				e, retriable, err := w.convToEvaluationEvent(ctx, evt, id, environmentNamespace)
 				if err != nil {
+					// If there is nothing to link, we don't report it as an error
+					handledCounter.WithLabelValues(codeNoLink).Inc()
```

Comment:
Can you also add `Linked` metrics?

E.g.
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/eventpersisterops/persister/evaluation_event.go#L90
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/eventpersisterops/persister/goal_event.go#L97